anomaly-detection/src/main/resources/app.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "anomaly-detection",
+  "producer": {
+    "latencyInMilliSec": 2000,
+    "capacity": 100,
+    "errorProbFactor": 0.3,
+    "controlledEnv": false
+  },
+  "pipeline": {
+    "parallelism": 1,
+    "checkpointIntervalInMilliSec": 1000,
+    "stateCheckpointDir": "hdfs://vijay-devbox:8020/checkpoint",
```

Comment:
Please remove reference to `vijay-devbox`.  Can't we just rely on the cluster's inbuilt configuration, and avoid setting the checkpoint directory in the app code?